src/test/app/Credentials_test.cpp

```diff
@@ -1058,6 +1058,43 @@ struct Credentials_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testFlags(FeatureBitset features)
+    {
+        using namespace test::jtx;
+
+        const char credType[] = "abcde";
```

Comment:
Need a `testcase` here

Reply:
it is on line  1079

Reply:
Typical convention involves having the `testcase` at the top of the test so it's easier to read.

Reply:
This function is reused for different tests, and testcase change it name

Reply:
Right, lines 1078-1081 can be moved up here

Reply:
moved